sample-input/benchmarks/c5g7/c5g7-3d-rodded-B.py

```diff
@@ -0,0 +1,526 @@
+import openmoc
+
+###############################################################################
+#######################   Main Simulation Parameters   ########################
+###############################################################################
+
+options = openmoc.options.Options()
+
+num_threads = options.getNumThreads()
+azim_spacing = options.getTrackSpacing()
+num_azim = options.getNumAzimAngles()
+tolerance = options.getTolerance()
+max_iters = options.getMaxIterations()
+
+openmoc.log.set_log_level('NORMAL')
+
+openmoc.log.py_printf('TITLE', \
+                      'Simulating the OECD\'s C5G7 Benchmark Problem...')
+
+
+###############################################################################
+###########################   Creating Materials   ############################
+###############################################################################
+
+openmoc.log.py_printf('NORMAL', 'Importing materials data from py...')
+
+materials = openmoc.materialize.materialize('../../c5g7-materials.py')
```

Comment:
Using the Python dictionary of materials rather than HDF5 for any particular reason? The Python dict support for materials is really only there for weird machines which don't support `h5py`